target/linux/realtek/dts/rtl9302_plasmacloud_psx8.dtsi

```diff
@@ -0,0 +1,271 @@
+				compatible = "openwrt,uimage", "denx,uimage";
+				openwrt,ih-magic = <0x93000000>;
+			};
+
+			partition@1080000 {
+				label = "firmware2";
+				reg = <0x1080000 0xf80000>;
+				compatible = "openwrt,uimage", "denx,uimage";
+				openwrt,ih-magic = <0x93000000>;
+			};
+		};
+	};
+};
+
+&ethernet0 {
+	nvmem-cells = <&macaddr_ubootenv_ethaddr>;
+	nvmem-cell-names = "mac-address";
+
+	mdio: mdio-bus {
+		compatible = "realtek,rtl838x-mdio";
+		regmap = <&ethernet0>;
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		/* External RTL8224 PHY */
+		phy0: ethernet-phy@0 {
+			reg = <0>;
+			compatible = "ethernet-phy-ieee802.3-c45";
+			rtl9300,smi-address = <0 0>;
+			sds = < 2 >;
+		};
+
+		phy1: ethernet-phy@1 {
+			reg = <1>;
+			compatible = "ethernet-phy-ieee802.3-c45";
+			rtl9300,smi-address = <0 1>;
+			sds = < 2 >;
+		};
+
+		phy2: ethernet-phy@2 {
+			reg = <2>;
+			compatible = "ethernet-phy-ieee802.3-c45";
+			rtl9300,smi-address = <0 2>;
+			sds = < 2 >;
+		};
+
+		phy3: ethernet-phy@3 {
+			reg = <3>;
+			compatible = "ethernet-phy-ieee802.3-c45";
+			rtl9300,smi-address = <0 3>;
+			sds = < 2 >;
+		};
+
+		phy8: ethernet-phy@8 {
+			reg = <8>;
+			compatible = "ethernet-phy-ieee802.3-c45";
+			rtl9300,smi-address = <0 4>;
+			sds = < 3 >;
+		};
+
+		phy9: ethernet-phy@9 {
+			reg = <9>;
+			compatible = "ethernet-phy-ieee802.3-c45";
+			rtl9300,smi-address = <0 5>;
+			sds = < 3 >;
+		};
+
+		phy10: ethernet-phy@10 {
+			reg = <10>;
+			compatible = "ethernet-phy-ieee802.3-c45";
+			rtl9300,smi-address = <0 6>;
+			sds = < 3 >;
+		};
+
+		phy11: ethernet-phy@11 {
+			reg = <11>;
+			compatible = "ethernet-phy-ieee802.3-c45";
+			rtl9300,smi-address = <0 7>;
+			sds = < 3 >;
+		};
+
+	};
+};
+
+&switch0 {
+	ports {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		port@0 {
+			reg = <0>;
+			label = "lan1";
+			phy-mode = "usxgmii";
+			phy-handle = <&phy0>;
+			led-set = <0>;
+		};
+
+		port@1 {
+			reg = <1>;
+			label = "lan2";
+			phy-mode = "usxgmii";
+			phy-handle = <&phy1>;
+			led-set = <0>;
+		};
+
+		port@2 {
+			reg = <2>;
+			label = "lan3";
+			phy-mode = "usxgmii";
+			phy-handle = <&phy2>;
+			led-set = <0>;
+		};
+
+		port@3 {
+			reg = <3>;
+			label = "lan4";
+			phy-mode = "usxgmii";
+			phy-handle = <&phy3>;
+			led-set = <0>;
+		};
+
+		port@8 {
+			reg = <8>;
+			label = "lan5";
+			phy-mode = "usxgmii";
+			phy-handle = <&phy8>;
+			led-set = <0>;
+		};
+
+		port@9 {
+			reg = <9>;
+			label = "lan6";
+			phy-mode = "usxgmii";
+			phy-handle = <&phy9>;
+			led-set = <0>;
+		};
+
+		port@10 {
+			reg = <10>;
+			label = "lan7";
+			phy-mode = "usxgmii";
+			phy-handle = <&phy10>;
+			led-set = <0>;
+		};
+
+		port@11 {
+			reg = <11>;
+			label = "lan8";
+			phy-mode = "usxgmii";
+			phy-handle = <&phy11>;
+			led-set = <0>;
+		};
+
+		port@28 {
+			ethernet = <&ethernet0>;
+			reg = <28>;
+			phy-mode = "internal";
+			fixed-link {
+				speed = <1000>;
+				full-duplex;
+			};
+		};
```

Comment:
When https://github.com/openwrt/openwrt/pull/19772 is merged earlier ("led-set" can be skipped because it is 0 by default):

```suggestion
		SWITCH_PORT(0, 1, usxgmii)
		SWITCH_PORT(1, 2, usxgmii)
		SWITCH_PORT(2, 3, usxgmii)
		SWITCH_PORT(3, 4, usxgmii)
		SWITCH_PORT(8, 5, usxgmii)
		SWITCH_PORT(9, 6, usxgmii)
		SWITCH_PORT(10, 7, usxgmii)
		SWITCH_PORT(11, 8, usxgmii)

		/* CPU-port */
		port@28 {
			ethernet = <&ethernet0>;
			reg = <28>;
			phy-mode = "internal";
			fixed-link {
				speed = <1000>;
				full-duplex;
			};
		};
```